lib/query-interface.js

```diff
@@ -638,7 +638,17 @@ class QueryInterface {
   }
 
   increment(instance, tableName, values, identifier, options) {
-    const sql = this.QueryGenerator.incrementQuery(tableName, values, identifier, options.attributes);
+    const sql = this.QueryGenerator.arithmeticQuery(' + ', tableName, values, identifier, options.attributes);
```

Comment:
we can run this without adding extra spaces, right.

If yes, we should not add any extra spaces. May be query generator should handle that if absolutely required, otherwise send query without spaces